### DISCUSSION TITLE
Week 2 Roundup

### DISCUSSION THREAD

**Discussion by DotNetDublin** [BlazingSaddlesSolutions/TimeKeeper#44] [General]
All in all another busy week for the community project and we even have some code to look at now 🙂

There's been some really interesting discussions and it's been a great introduction on working in an open source project.

We've simplified the starting code in order to make it easier for people to review and offer alternative project architecture options and this is what we will probably focus on next week.

Finally I may have fallen into the "well it works on my machine" trap so if you clone the repository and can't get it working bear with us, check out the read me and raise an issue.

**Comment by ripteqdavid** [comment]
Wow!  I have one long weekend away from the internet and this happens!  What amazing progress.  I can't wait to dig in and wrap my head around the discuissions.

**Reply by DotNetDublin** [reply]
Great. If you could agree with all my points in the discussions that would be swell 😄

**Reply by ripteqdavid** [reply]
I'll get the 'I agree with Francisco' copy-paste shortcut ready :)
